services/manifesto_model/api.py

```diff
@@ -117,6 +118,41 @@ def prioritized_texts():
     # print(texts_priotized)
     return jsonify({'data': texts_priotized[:n_texts]})
 
+@app.route("/prioritized_texts_with_label", methods=['GET'])
+def prioritized_texts_with_label():
+    """
+    Retrieves all of the political statements from the database, prioritizes them with the manifesto model
+    and returns at most as many texts given in the GET parameter `n`, ordered by their uncertainty.
+
+    Half of the texts have known labels
+
+    :return: {
+        'data': [
+            {'text_id': 1, 'label': 'left'},
+            ...
+        ]
+    }
+    """
+    print("getting prioritized texts")
+    n_texts = int(request.args.get('n'))
+    texts = get_texts_only(n_all_samples)
+    prios_texts = classifier.prioritize(map(lambda t: t['statement'], texts))
+    # print(prios_texts)
+    texts_prioritized = np.array(texts)[np.array(prios_texts)].tolist()
+    to_label = texts_prioritized[:int(n_texts/2)]
+    for sample in to_label:
+        sample['label'] = ""
+
+    texts_with_labels = random.sample([sample for sample in
+        get_texts_with_labels(100, "majority")
```

Comment:
Einmal holen wir n Texte und ein andermal 100. Warum?

Reply:
Also ich meine, ist es beabsichtigt?

Reply:
ja, das is bisschen willkuerlich, aber ich wollte keine neutral texte zeigen, und left/right sind nur weniger als 30 %, deswegen dachte ich sind bei 100 im mittel genug links rechts dabei.